static/js/components/dashboard/CourseRow_test.js

```diff
@@ -42,11 +42,11 @@ describe('CourseRow', () => {
         prices={prices}
         openFinancialAidCalculator={sandbox.stub()}
         now={moment()}
-        checkout={sandbox.stub()}
         addCourseEnrollment={sandbox.stub()}
         course={null}
         {...props}
-      />
+      />,
+      {context: { router: {}}, childContextTypes: {router:   React.PropTypes.object.isRequired}}
```

Comment:
Can you reformat this dict like this?

    {
      context: {
        router: {}
      },
      childContextTypes: {
        router: React.PropTypes.object.isRequired
      }
    }